lib/onlineorder.rb

```diff
@@ -0,0 +1,71 @@
+require 'csv'
+require_relative 'order.rb'
+require_relative 'customer.rb'
+
+module Grocery
+	class OnlineOrder < Order
+		attr_reader :id, :products, :customer, :status
+
+		def initialize(id, products, customer, status='pending')
+			super(id, products) #call the Order's constructor and passes variables as needed
```

Comment:
Good work with this class - you've done a really nice job of using inheritance and `super` to DRY up this code.